### PR TITLE
Release Godot 4.1.2 RC1 and publish release notes

### PR DESCRIPTION
**Pull request by YuriSizov**

### CODE REVIEW

File: collections/_article/release-candidate-godot-4-1-2-rc-1.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
+---
+title: "Release candidate: Godot 4.1.2 RC 1"
+excerpt: "It's long overdue for the second Godot 4.1 patch release! This release candidate contains a number of important fixes, including performance and stability improvements, so let's give it a good test."
+categories: ["pre-release"]
+author: Yuri Sizov
+image: /storage/blog/covers/release-candidate-godot-4-1-2-rc-1.webp
+image_caption_title: "Fisk"
+image_caption_description: "A game by Kristian Nilsen"
+date: 2023-09-22 16:00:00
```
Comment (akien-mga) [inline collections/_article/release-candidate-godot-4-1-2-rc-1.md]:
```suggestion
date: 2023-09-22 14:00:00
```
Remember it's UTC.

File: collections/_article/release-candidate-godot-4-1-2-rc-1.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+
+Maintenance releases are expected to be safe for an upgrade, but we recommend to always make backups, or use a version control system such as Git, to preserve your projects in a case of corruption or data loss.
+
+[Jump to the **Downloads** section](#downloads), and give it a spin right now, or continue reading to learn more about improvements in this release. You can also [try the **Web editor**](https://editor.godotengine.org/releases/4.1.2.rc1/) or the **Android editor** for this release. If you are interested in the latter, please request to join [our testing group](https://groups.google.com/g/godot-testers) to get access to pre-release builds.
+
+-----
+
+*The illustration picture for this article comes from* [**Fisk**](http://fiskthegame.com/) *— an open world survival horror game with Lovecraftian vibes currently being developed by [Kristian Nilsen](https://twitter.com/sajbear666) with Godot 4. (It uses* [**Qodot**](https://github.com/QodotPlugin/Qodot/) *for creating geometry, by the way, which deserves its own shoutout!) You can follow Kristian on [Twitter](https://twitter.com/sajbear666) for more updates and you can download Fisk today from the website.*
```
Comment (akien-mga) [inline collections/_article/release-candidate-godot-4-1-2-rc-1.md]:
```suggestion
*The illustration picture for this article comes from* [**Fisk**](http://fiskthegame.com/) *— an open world survival horror game with Lovecraftian vibes currently being developed by [Kristian Nilsen](https://twitter.com/sajbear666) with Godot 4. (It uses* [**Qodot**](https://github.com/QodotPlugin/Qodot/) *for creating geometry, by the way, which deserves its own shoutout!) You can follow Kristian on [Twitter](https://twitter.com/sajbear666) for more updates and you can download Fisk today from [its website](http://fiskthegame.com/).*
```

**Review by akien-mga — APPROVED** [review]

**Review by adamscott — APPROVED** [review]
Everything seems fine, Sir.

File: collections/_article/release-candidate-godot-4-1-2-rc-1.md (hunk 1 above)
Comment (YuriSizov) [reply collections/_article/release-candidate-godot-4-1-2-rc-1.md]: Yeah, I just give it an arbitrary estimate when I start :) We managed to get ready quicker!

**Comment by YuriSizov** [comment]
Thanks for reviews!

**Pull request merged**